### PR TITLE
feat(scan): bridge vector tiles and feature services

### PR DESCRIPTION
**Pull request by ibgreen**
## Goals

- Land the first conclusive tiles/services bridge while keeping physical tile and service controls outside the portable `TableQuery` AST.
- Let MVT, vector PMTiles, WFS, and ArcGIS FeatureServer results reuse the shared Arrow scan/query surface through the opt-in `@loaders.gl/scan` package.
- Preserve lightweight format imports, truthful residual-query diagnostics, and cooperative cancellation.

## Changes

- Add `VectorTileTableScanSource` and `VectorFeatureTableScanSource` adapters that bind one physical tile or bounded service request, then expose metadata, query, explain, and read operations over the resulting Arrow feature table.
- Cache the addressed Arrow result across metadata discovery and query execution, infer feature-column roles and spatial metadata, and reject null or non-Arrow source output with actionable errors.
- Add `AbortSignal` to `GetTileParameters`, forward it through MVT tile fetches, and remove PMTiles' untyped signal access.
- Correct the Arrow executor's existing explain path so it reports residual operators consistently with `ARROW_TABLE_QUERY_CAPABILITIES` instead of SQL-source pushdown.
- Document addressed vector tables and update roadmap tranche 7 and the end-user support scorecard.
- Add owning and integration tests for addressing boundaries, metadata, bounds, caching, schema inference, filtering, projection, limit, explain, cancellation, invalid output, and retry behavior.

## Validation

- `yarn lint fix`
- `yarn build`
- `yarn build-workers`
- `yarn test-audit`
- `yarn test-node` — 360 passed, 6 skipped
- `yarn test-headless` — 3009 passed, 40 skipped
- focused post-rebase Chromium suite — 29 passed
- full Node + Chromium coverage and Chromium-only coverage
- `yarn test-coverage-check` — 53 packages, 0 violations; new scan adapter at 100% statements, lines, functions, and branches

### CODE REVIEW

File: modules/scan/src/vector-table-scan-source.ts [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+ */
+export class VectorFeatureTableScanSource extends AddressedVectorTableScanSource {
+  /** Specialized vector source that owns service request construction and parsing. */
+  readonly source: VectorSource;
+  /** Immutable bounded feature request bound to this table view. */
+  readonly request: Omit<GetFeaturesParameters, 'format' | 'signal'>;
+
+  /** Creates a portable table view over one bounded vector-service request. */
+  constructor(source: VectorSource, options: VectorFeatureTableScanSourceOptions) {
+    const request = Object.freeze({...options.request});
```
[the remark this answers is not in the document]
Comment (ibgreen) [reply modules/scan/src/vector-table-scan-source.ts]: Fixed in 3f7d0b2375. Bound tile and feature requests now clone and freeze layer arrays, and feature bounds clone/freeze both coordinate tuples plus the outer array. Regression tests mutate the original inputs and verify request and metadata remain aligned.

File: modules/scan/src/vector-table-scan-source.ts [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+        coordinateReferenceSystems: ['EPSG:4326']
+      }
+    });
+    this.source = source;
+    this.tile = tile;
+  }
+
+  /** Loads the bound vector tile using the source's requested Arrow output shape. */
+  protected loadTable(signal?: AbortSignal): Promise<ArrowTable | null> {
+    return this.source.getVectorTile({...this.tile, signal}) as Promise<ArrowTable | null>;
```
[the remark this answers is not in the document]
Comment (ibgreen) [reply modules/scan/src/vector-table-scan-source.ts:189]: Fixed in 3f7d0b2375. MVTTileSource and PMTilesTileSource now forward the bound non-empty tile layer selection into the MVT decoder while preserving inherited loader options; focused regression tests cover both sources and the empty-selection fallback.

File: modules/scan/src/vector-table-scan-source.ts [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+      data: result.data,
+      length: result.data.numRows
+    };
+  }
+
+  /** Resolves and caches the addressed table while allowing failed loads to be retried. */
+  private async getTable(signal?: AbortSignal): Promise<ArrowTable> {
+    throwIfAborted(signal);
+    if (!this.tablePromise) {
+      this.tablePromise = this.loadTable(signal)
```
[the remark this answers is not in the document]
Comment (ibgreen) [reply modules/scan/src/vector-table-scan-source.ts]: Fixed in 3f7d0b2375. The cached physical load is no longer owned by the first caller signal; each consumer now independently races its wait against its own AbortSignal. A concurrency regression verifies one physical load, prompt rejection for the cancelled caller, and success for the uncancelled caller.
